Library/Homebrew/dev-cmd/tests.rb

```diff
@@ -82,6 +82,7 @@ def tests
         "--format", "ParallelTests::RSpec::RuntimeLogger",
         "--out", "tmp/parallel_runtime_rspec.log"
       ]
+      spec_args << "--tag" << "~needs_official_cmd_taps" if ENV["HOMEBREW_TEST_OFFICIAL_CMD_TAPS"].nil?
```

Comment:
What does this do? Avoid them being skipped or skip them with nicer syntax? If the latter: I'm not sure I agree. If the prior: I think it's actually worth them being noted as skipped so people are aware they aren't running a full test.